docs/installation/newer-versions.csv

```diff
@@ -1,9 +1,10 @@
-﻿Python,3.13,3.12,3.11,3.10,3.9,3.8,3.7,3.6,3.5
-Pillow >= 11,Yes,Yes,Yes,Yes,Yes,,,,
-Pillow 10.1 - 10.4,,Yes,Yes,Yes,Yes,Yes,,,
-Pillow 10.0,,,Yes,Yes,Yes,Yes,,,
-Pillow 9.3 - 9.5,,,Yes,Yes,Yes,Yes,Yes,,
-Pillow 9.0 - 9.2,,,,Yes,Yes,Yes,Yes,,
-Pillow 8.3.2 - 8.4,,,,Yes,Yes,Yes,Yes,Yes,
-Pillow 8.0 - 8.3.1,,,,,Yes,Yes,Yes,Yes,
-Pillow 7.0 - 7.2,,,,,,Yes,Yes,Yes,Yes
+﻿Python,3.14,3.13,3.12,3.11,3.10,3.9,3.8,3.7,3.6,3.5
+Pillow 12,Yes,Yes,Yes,Yes,Yes,,,,,
+Pillow 11,,Yes,Yes,Yes,Yes,Yes,,,,

```

Comment:
```suggestion
Pillow >= 12,Yes,Yes,Yes,Yes,Yes,,,,,
Pillow 11.0 - 11.3,,Yes,Yes,Yes,Yes,Yes,,,,

```

Reply:
11.0-11.3 is all the 11 series releases.

Most other major series have been split over Python support, but the annual release cycle has now aligned so we can just say 11, and our readers don't need to consider "was there a 11.4?"

Similar to Pillow 4 (and older):

https://pillow.readthedocs.io/en/stable/installation/python-support.html